.github/workflows/release.yml

```diff
@@ -140,13 +140,22 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: google-github-actions/auth@v0
+        continue-on-error: true
+        with:
+          credentials_json: ${{ secrets.gcs_bazel_cache }}
       - name: Build macOS wheels
         run: |
           python --version
           python -m pip install delocate wheel setuptools numpy six --no-cache-dir
 
           ./configure.py
 
+          if [[ -n $GOOGLE_APPLICATION_CREDENTIALS ]]; then
```

Comment:
For the release workflow this should always exist, but I agree it's probably good to keep this check just to be safe.